test/e2e/specs/editor/blocks/details.spec.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Details', () => {
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
+	test( 'can toggle hidden blocks by pressing enter', async ( {
+		editor,
+		page,
+	} ) => {
+		// Insert a details block with empty inner blocks.
+		await editor.insertBlock( {
+			name: 'core/details',
+			attributes: {
+				summary: 'Details summary',
+			},
+		} );
```

Comment:
```suggestion
		await editor.insertBlock( {
			name: 'core/details',
			attributes: {
				summary: 'Details summary',
			},
			innerBlocks: [
				{
					name: 'core/paragraph',
					attributes: { content: 'Details content' },
				},
			],
		} );
```

It might be better to use an inner block with explicit text.